enhancements/samples/removed-bootstrapping.md

```diff
@@ -0,0 +1,203 @@
+skip list in the operator config object
+- override the registry used by the samples operator in its config object
+- or if you want to punt on samples entirely, mark the operator as `Removed`
+
+### Goals
+
+1) Alleviate out of the install concerns in scenarios where samples installation will fail out of the gate
+
+2) Allow the cluster administrator time to decide what to do with samples in the cluster
+
+### Non-Goals
+
+Other usability improvements around installing samples from registries other than registry.redhat.io have 
+been identified in Jira:
+
+- regular expressions for further manipulation of the image reference in the imagestream
+- modes where failed imagestreams are automatically added to the skip list (where the administrator then
+automatically can have them added to the management set by fixing the import by mirroring the images, fixing
+the registry in question, etc.)
+- moving the samples operator out of the payload / CVO and making it an OLM operator, where the cluster
+administrator controls which imagestreams are installed when the provision the OLM operator
+
+Bootstrapping as removed is considered the meets minimum, least impacting change to the samples operator 
+that addresses the scenarios where out of the box imagestreams pointing to registry.redhat.io will fail.
+
+## Proposal
+
+Out startup today, the samples operator bootstraps as `Managed`.  That boostrapping can be made conditional,
+and instead come up as `Removed` when the samples operator sees
+
+- Via the techniques noted in the "OCP 4.3 single stack IPv6 control stack" document at
+https://docs.google.com/document/d/1zznvkK87y7UvOJZqs2DM4oWzx94P8Lg6YUb43whil3g/edit#heading=h.q0ahufbae2kz the 
+samples operator determines that it is in an IPv6 single stack scenario
+- By inspecting the image of its own pod, if the samples operator sees something other than 
+`registry.svc.ci.openshift.org` or `registry.redhat.io` assume it is a disconnected / restricted network install
+scenario
+- Or (or additionally) attempt to create TCP connections to `registry.redhat.io` over the course of 1 minute and 
+if those fail, assume it is a disconnected / restricted network install scenario 
+
+### User Stories [optional]
+
+#### Story 1
+
+#### Story 2
+
+### Implementation Details/Notes/Constraints [optional]
+
+### Risks and Mitigations
+
+1) If the pod or api object inspections noted in the proposal are not definitive and misidentify some percentage of
+the scenarios 
+
+## Design Details
+
+### Test Plan
+
+Consider the following in developing a test plan for this enhancement:
+- QE already had some disconnected CI and we've been working with them via bugzillas to get it set up.  This change
+could remove the need for additional administration on their part to keep the cluster from having `Degraded` operators
+- For IPv6, the samples operator e2e will be updated to inspect the test pod for IPv6 and adjust accordingly
```

Comment:
presumably more than just the samples operator e2e will need adjustment for CI to pass on ipv6 since no samples are going to import.  But that's true whether or not you make this change since the samples already don't import today.

Reply:
yep chose not to speak to broader concerns ... I'll keep this as is